deploy/helm/trickster/values.yaml

```diff
@@ -3,14 +3,28 @@
 # Default trickster originURL, references a source Prometheus instance
 # Ref: https://github.com/Comcast/trickster/blob/master/docs/configuring.md
 originURL: http://prometheus:9090
+cacheType: memory
```

Comment:
I'm thinking this might be a better way to structure these cache related variables
```
cache:
  type: memory
  redis:
    protocol: tcp
    endpoint: redis:6379
  filesystem:
    path: /tmp/trickster
```

Reply:
Sure, that's fine with me.  Let me change it quickly.